### DISCUSSION TITLE
Expressvpn VPN in use. Does not work normally. An error has occurred.

### DISCUSSION THREAD

**Discussion by LanGsA0** [AdguardTeam/AdGuardHome#2955] [Q&A]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [V] I am running the latest version
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * 
* **How did you install AdGuard Home:**
  * 
* **How did you setup DNS configuration:**
  * 
* **If it's a router or IoT, please write device model:**
  * 
* **CPU architecture:**
  * 
* **Operating system and version:**
  * 

### Screenshots

<details><summary>Screenshot:</summary>
![image](https://user-images.githubusercontent.com/70998972/113957008-70634f80-9859-11eb-8c16-94a44385a9d7.png)

</details>

**Comment by ainar-g** [comment]
It looks like AGH cannot reach any of the bootstrap servers.  Please fill the Issue Details section properly and also make sure that your system allows outgoing UDP connections to port 53.
